product/views.py

```diff
@@ -1,3 +1,92 @@
-from django.shortcuts import render
+import json
+import random
+
+from django.views import View
+from django.http  import (
+HttpResponse,
+JsonResponse
+)
+
+from my_settings import (
+    SECRET_KEY, 
+    ALGORITHM
+)
+
+from user.models import User
+from .models     import (
+Product,
+Applying,
+Category,
+Color,
+Image,
+ImageCategory
```

Comment:
파이썬 기본 indent 는 4칸 입니다.
정리 부탁 드리겠습니다.
```suggestion
    Product,
    Applying,
    Category,
    Color,
    Image,
    ImageCategory
```